### PR TITLE
Add spaces in comments

### PR DESCRIPTION
**Pull request by samanthamjohn**
## WHAT

Make spacing more consistent with the following cops: 
- [Layout/LeadingCommentSpace](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/LeadingCommentSpace)
- [Layout/SpaceAfterComma](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceAfterComma)
- [Layout/SpaceAroundEqualsInParameterDefault](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceAroundEqualsInParameterDefault)
- [Layout/SpaceAroundOperators](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceAroundOperators)
- [Layout/SpaceBeforeBlockBraces](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceBeforeBlockBraces)

## WHY
- These should be relatively uncontroversial as they're just spacing rules and nothing came out looking very weird. 
- Fixing the layout caused the security bot to find issues that it did not notice before. 
- This frees up developer brain space to focus on other parts of code quality rather than formatting. 

## HOW
Remove disabled flags from the rubocop todo file. run rubop -A

### What have you done to QA this feature?
ran tests
PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  no
Have you deployed to Staging? | no
Self-Review: Have you done an initial self-review of the code below on Github? | YES

### CODE REVIEW

File: services/QuillLMS/app/controllers/schools_controller.rb [hunk 1]
```diff
@@ -122,7 +122,7 @@
     array_encoder = PG::TextEncoder::Array.new
     literal_encoder = PG::TextEncoder::QuotedLiteral.new
     r = array_encoder.encode(ruby_array.map { |v| literal_encoder.encode(v) })
-    r.sub('{','(').sub('}', ')')
+    r.sub('{', '(').sub('}', ')')
```
[the remark this answers is not in the document]
Comment (brendanshean) [reply services/QuillLMS/app/controllers/schools_controller.rb]: Maybe a `gsub` here?

[the remark this answers is not in the document]
Comment (brendanshean) [reply services/QuillLMS/app/controllers/schools_controller.rb]: Maybe a `gsub` here?

**Comment by brendanshean** [comment]
> Make spacing more consistent with the following cops:
> 
> * [Layout/LeadingCommentSpace](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/LeadingCommentSpace)
> * [Layout/SpaceAfterComma](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceAfterComma)
> * [Layout/SpaceAroundEqualsInParameterDefault](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceAroundEqualsInParameterDefault)
> * [Layout/SpaceAroundOperators](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceAroundOperators)
> * [Layout/SpaceBeforeBlockBraces](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceBeforeBlockBraces)
>   I think they should be relatively uncontroversial. Also, fixing the layout caused the security bot to find issues that it did not notice before. I think we can dismiss those alerts for now, though maybe we'd want to fix in the future.

Let's use the PR template.

**Comment by samanthamjohn** [comment]
> Let's use the PR template.

Sorry, had this idea to do a bunch of really small PRs with smaller messages. then decided to put them together in a logical way and the PR got bigger.

**Review by brendanshean — APPROVED** [review]
Thank you for adding these cops in! 😄

**Review by anathomical — APPROVED** [review]

**Pull request merged**